src/components/modal/Modal.vue

```diff
@@ -133,6 +135,11 @@ export default {
     watch: {
         active(value) {
             this.isActive = value
+            this.$nextTick(() => {
+                if (value && this.$el && this.$el.focus) {
+                    this.$el.focus()
+                }
+            })
```

Comment:
This is to give the focus to the element. If we dont do this, once we open the modal and we use `TAB`, the focus is not given within the modal. Take this eample:
https://codepen.io/dalton/pen/BaaYNGJ